CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - A more compact case page
 - Removed OMIM genes panel
 - Make genes panel, pinned variants panel, causative variants panel and ClinVar panel scrollable on case page
+- Update to Scilifelab's 2020 logo
```

Comment:
Is it from last year or this year?

Reply:
It was launched on the 10th of September 2020 (https://www.scilifelab.se/community-pages/news/meet-our-new-logo/).